examples/k8petstore/k8petstore.sh

```diff
@@ -46,173 +46,193 @@ function create {
 
 cat << EOF > fe-rc.json
 {
-  "id": "fectrl",
   "kind": "ReplicationController",
-  "apiVersion": "v1beta1",
-  "desiredState": {
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "fectrl",
+    "labels": {"name": "frontend"}
+  },
+  "spec": {
     "replicas": $FE,
-    "replicaSelector": {"name": "frontend"},
-    "podTemplate": {
-      "desiredState": {
-         "manifest": {
-           "version": "v1beta1",
-           "id": "frontendCcontroller",
-           "containers": [{
-             "name": "frontend-go-restapi",
-             "image": "jayunit100/k8-petstore-web-server:$VERSION"
+    "selector": {"name": "frontend"},
+    "template": {
+      "spec": {
+         "containers": [{
+           "name": "frontend-go-restapi",
+           "image": "jayunit100/k8-petstore-web-server:$VERSION"
          }]
-         }
        },
-       "labels": {
-         "name": "frontend",
-         "uses": "redis-master"
+       "metadata": {
```

Comment:
@pedro-r-marques, please remove the name fields of other pod templates as well, thanks.